Marlin/src/module/motion.cpp

```diff
@@ -966,6 +966,48 @@ inline float get_homing_bump_feedrate(const AxisEnum axis) {
   return homing_feedrate(axis) / hbd;
 }
 
+#if ENABLED(SENSORLESS_HOMING)
+
+  /**
+   * Set sensorless homing if the axis has it, accounting for Core Kinematics.
+   */
+  void sensorless_homing_per_axis(const AxisEnum axis, const bool enable/*=true*/) {
```

Comment:
Don't you need to propergate enable to tmc_sensorless_homing?

Reply:
Omigod, yes! I was hopefully going to catch that once I got home and re-checked. Thanks for your good eye!